recipe/meta.yaml

```
@@ -51,18 +51,16 @@ requirements:
     - packaging >=20.4
     - pexpect >=4.7.0,<5.0.0
     - pkginfo >=1.9.4,<2.0
-    - platformdirs >=2.5.2,<3.0.0
+    - platformdirs >=3.0.0,<4.0.0
```

Was it deliberate to leave out `backports.cached-property`? Otherwise it may be broken on Python 3.7.

```suggestion
    - platformdirs >=3.0.0,<4.0.0
    - backports.cached-property >=1.0.2,<2.0.0
```

---

We don't support python 3.7 on conda-forge anymore and we want to stay noarch: python.  But since we're noarch: python maybe best to add it.

---

Ya, let's either add it or alternatively we could raise the lower pin on `python` to `>=3.8`.

---

I added it.  There may still be folks wanting to use poetry with python 3.7 and no reason we can't let them at least try.  I bet the dependencies won't be there, though.

---

BTW, this was the same for the last release and no one complained so I don't think anyone is trying to use python 3.7 (or at least they didn't notice the issue if they did).